composer.json

```diff
@@ -24,7 +24,7 @@
         "kriswallsmith/buzz": ">=0.9",
         "tedivm/stash-bundle": "0.4.*",
         "sensio/distribution-bundle": "~3.0",
-        "nelmio/cors-bundle": "1.3.*",
+        "nelmio/cors-bundle": "1.4.*",
```

Comment:
Maybe rather change to `^1.3.3`, also allowing 1.3.3 as long as we don't use any features of 1.4 yet.